gradle.properties

```diff
@@ -1,6 +1,6 @@
 org.gradle.jvmargs=-Xmx3G
 modGroup=me.zeroeightsix
-modVersion=1.1.7-beta
+modVersion=1.11.xx-dev
```

Comment:
Why is the month hard coded?, also instead of using magical number at the beginning, it is better to use YY (20 for 2020, 21 for 2021)

Reply:
This isn't hardcoded, and the "magical number" isn't magical, it's bumped automatically by 1 each year